packages/app/src/cli/models/app/app.test-data.ts

```diff
@@ -39,6 +39,7 @@ export async function testUIExtension(uiExtension: Partial<UIExtension> = {}): P
     capabilities: {
       block_progress: false,
       network_access: false,
+      storefront_api: false,
```

Comment:
I think we had a thread going about making this be `api_access` instead of `storefront_api` — where did we land on that?

Reply:
Updated now